octoprint_mrbeam/templates/settings/about_settings.jinja2

```diff
@@ -79,7 +79,7 @@
 
                     <div><strong>{{ _('Online Support Portal') }}:</strong> <a href="https://www.mr-beam.org/support" target="_blank">mr-beam.org/support</a></div>
                     <div><strong>{{ _('Privacy Policies') }}:</strong>
-                        <a href="http://shop.mr-beam.org/privacy" target="_blank">{{ _('Mr Beam Web Site') }}</a> |
+                        <a href="{{ _('https://www.mr-beam.org/en') + '/policies/privacy-policy'}}" target="_blank">{{ _('Mr Beam Web Site') }}</a> |
```

Comment:
Are we always redirecting to the English page?

Reply:
Yes, all links are added for the English version by default. Later in another ticket upon translating these strings to German we only have to remove the /en for the link to show the German version of the page